rest-api-spec/src/main/resources/rest-api-spec/test/search_pipeline/10_basic.yml

```diff
@@ -32,8 +32,8 @@
 ---
 "Test Put Versioned Pipeline":
   - skip:
-      version: " - 2.99.99"
-      reason: "to be introduced in future release / TODO: change if/when we backport to 2.x"
+      version: " - 2.7.0"
+      reason: "Added in 2.7.0"
```

Comment:
Should this be the following?

```
version: " - 2.7.99"
reason: "Added in 2.8.0"
```

Reply:
These APIs were available in 2.7.0 (behind the feature flag).

I just never updated this condition after backporting.